src/ontology/cl.Makefile

```diff
@@ -20,6 +20,11 @@
 #tmp/pr_logical.owl: mirror/pr.owl
 #	echo "Skipped pr logical" && cp $< $@
 
+.PHONY: rdiff
+rdiff: 
+	curl https://raw.githubusercontent.com/shawntanzk/cell-ontology/master/src/ontology/cl-edit.owl -o rdifftmp.owl
```

Comment:
best do not curl (unnecessary dependency on network - always avoid), use `checkout master -- $(SRC)` instead.

Reply:
thanks! this was the part that I was quite unhappy about how inelegant it was, i usually use git show for this part but didnt know the best way to do this otherwise :)

Reply:
git show may be fine, maybe even better, I have not used it - just do not curl!

Reply:
the struggle I had with that was that I couldn't get git commands to work with actions - trying this on test fork and it seems to be failing there too. but 100% agree that curl is not ideal

Reply:
I think the cleanest thing is to orchestrate this in the action itself, have multiple steps with different checkouts, each generating the file in question, then running the diff in a third step.

Reply:
@shawntanzk here's what I implemented for GO a while back, maybe it can be of use: https://github.com/geneontology/go-ontology/blob/master/.github/workflows/inference_diff.yml

Reply:
Oh YES, that is right you did that.. Sorry man I forgot about it. I think this is the best design!

Reply:
sweet! thanks so much @balhoff :D

Reply:
Do we need 4 jobs, classify_pr, classify_master, diff_classification and post_comment? We are checking out our branch 3 times is that necessary? Can I use single job for, downloading branch, master, doing diff analysis and posting a comment? Or is that a bad design? @matentzn

Reply:
The separate jobs made it easier (to me) to work on two different branches. It could be 3 total jobs, but the diff and the comment used different docker images, so they are different jobs. But it sounds like you found a better comment posting action. Also with multiple jobs, the two classifications can happen simultaneously.